docs/getting_started/hopsworksai/guides/eks_ecr_integration.rst

```diff
@@ -0,0 +1,327 @@
+
+.. code-block:: bash
+    
+    eksctl utils describe-stacks --region=us-east-2 --cluster=my-eks-cluster | grep 'OutputKey: "ClusterSecurityGroupId"' -a1
+
+Check the output for *OutputValue*, that will be the id of your eks security group.
+
+.. code-block:: bash
+
+    ExportName: "eksctl-my-eks-cluster-cluster::ClusterSecurityGroupId",
+    OutputKey: "ClusterSecurityGroupId",
+    OutputValue: "YOUR_EKS_SECURITY_GROUP_ID"
+
+Once you get the security group id (YOUR_EKS_SECURITY_GROUP_ID), you need to proceed to the AWS management console by clicking `security groups <https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#SecurityGroups:>`_. Filter security groups using the *Security Group ID* and then paste your eks security group id. Click on the *inbound rules* tab, then click on the *Edit inbound rules*, now you should arrive at the following screen.
+
+.. _sg-edit-inbound-rules.png: ../../../_images/eks-sg-edit-inbound-rules.png
+.. figure:: ../../../imgs/hopsworksai/eks-sg-edit-inbound-rules.png
+    :alt: Edit inbound rules
+    :target: `sg-edit-inbound-rules.png`_
+    :align: center
+    :figclass: align-center
+
+Add two rules for HTTP and HTTPS as follows:
+
+.. _sg-hopsworks-inbound-rules.png: ../../../_images/eks-sg-hopsworks-inbound-rules.png
+.. figure:: ../../../imgs/hopsworksai/eks-sg-hopsworks-inbound-rules.png
+    :alt: Edit inbound rules
+    :target: `sg-hopsworks-inbound-rules.png`_
+    :align: center
+    :figclass: align-center
+
+Click *Save rules* to save the updated rules to the security group.
+
+Step 5: Create a hopsworks cluster with EKS and ECR support
+-----------------------------------------------------------
+
+In Hopsworks.ai, select *Create cluster*. Choose the region of your EKS cluster, then click Next:
+
+.. _eks-hopsworks-create-cluster.png: ../../../_images/eks-hopsworks-create-cluster.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster.png`_
+    :align: center
+    :figclass: align-center
+
+Choose your preferred SSH key to use with the cluster, then click Next:
+
+.. _eks-hopsworks-create-cluster-0.png: ../../../_images/eks-hopsworks-create-cluster-0.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-0.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-1.png`_
+    :align: center
+    :figclass: align-center
+
+Choose Enabled to enable the use Amazon EKS and ECR:
+
+.. _eks-hopsworks-create-cluster-1.png: ../../../_images/eks-hopsworks-create-cluster-1.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-1.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-1.png`_
+    :align: center
+    :figclass: align-center
+
+Add your EKS cluster name and update your ECR account id if you want to use another account for ECR, then click Next:
```

Comment:
I guess the `ECR account id` is what is called `AWS Account Id` in the picture? If it is the case we should use the same name in both places.